2015/presentations/with modern_peripherals: Python and Flask Piotr Dyba/text.md

```diff
@@ -0,0 +1,208 @@
+# with modern_peripherals: Python and Flask - Piotr Dyba
+
+##Introduction
+
+Auto-scrolling sites, glance-following ads, and gesture friendly web pages are 
+coming! Over the last few years three products emerged that enable interaction 
+with computer in a new way: Myo Armband, Leap Motion Controller and EyeTribe. 
+Using web sockets it is possible to interact using this new devices with web 
```

Comment:
How about: The integration between the aforementioned products and other devices and web apps is based on the web sockets protocol. So lets start with...

I think it's simpler, shows the relationship between products and web sockets and gets rid of duplicated "using" and "new way".